src/routes/open/container/Open.tsx

```diff
@@ -192,7 +191,18 @@ const Open = (): React.ReactElement => {
       action: 'Created a safe',
     })
 
-    removeFromStorage(SAFE_PENDING_CREATION_STORAGE_KEY)
+    // a default 5s wait before starting to request safe information
+    await sleep(5000)
+
+    await backOff(() => getSafeInfo(safeAddress), {
+      startingDelay: 750,
```

Comment:
Awesome! 👏 
Will the 10 default attempts be enough?

Reply:
From what I tested, yes. But it will let the button in an infinite "loading" state if we reach the maximum of attempts 🤔

Reply:
hm..., it will redirect to the `/balances` endpoint anyway. I believe that for the current state of affairs it is a, so far, good solution.